source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOAdaptiveCard.mm

```diff
@@ -16,6 +17,41 @@
 @implementation ACOAdaptiveCard
 {
     std::shared_ptr<AdaptiveCard> _adaptiveCard;
+    NSMutableArray *_inputs;
+}
+
+- (void) setInputs:(NSMutableArray *) inputs
+{
+    _inputs = inputs;
+}
+
+- (void) appendInputs:(NSMutableArray *)inputs
+{
+    [_inputs addObjectsFromArray:inputs];
+}
+
+- (NSData *)inputs:(NSError **)error
+{
+    if(_inputs){
+        NSMutableDictionary *dictionary = [[NSMutableDictionary alloc] init];
+        for(id<ACRIBaseInputHandler> input in _inputs)
+        {
+            if([input validate:error] == NO)
+            {
+                NSLog(@"input validation failed %@", *error);
```

Comment:
>NSLog(@"input validation failed %@", *error); [](start = 16, length = 45)

If multiple errors are encountered, the error returned to the caller will be whatever the last error was.

How is someone supposed to call this method? What if the method returns an NSData* and also an NSError*? Is that a success or a failure?

Reply:
we don't have to validate inputs anymore, i will remove them. thank you


---
In reply to: [172390450](https://github.com/Microsoft/AdaptiveCards/pull/1211#discussion_r172390450) [](ancestors = 172390450)